cmd/foreach/foreach.go

```diff
@@ -151,6 +185,27 @@ func setupOutputFiles(campaignName string, command string) {
 	defer successfulReposFile.Close()
 	defer failedReposFile.Close()
 
+	if _, err := os.Lstat(successfulReposSymlink); err == nil {
```

Comment:
You can prob just override it?

Reply:
os.Symlink seems to error if there is a previously existing symlink with the same name